.rubocop_todo.yml

```diff
@@ -92,13 +132,19 @@ Performance/CollectionLiteralInLoop:
     - 'lib/puppet-lint/plugins/check_resources/ensure_first_param.rb'
     - 'lib/puppet-lint/plugins/check_whitespace/trailing_whitespace.rb'
 
-# Offense count: 406
+# Offense count: 1
+# This cop supports unsafe autocorrection (--autocorrect-all).
+Performance/MapCompact:
+  Exclude:
+    - 'lib/puppet-lint/plugins/check_unsafe_interpolations/check_unsafe_interpolations.rb'
```

Comment:
Looks like you already fixed this, so perhaps this is gone if you regenerate the TODO file.

Reply:
good spot! Thanks